fpdf/fpdf.py

```diff
@@ -505,6 +507,15 @@ def __init__(
         # page number -> array of 8 × n numbers:
         self.text_quad_points = defaultdict(list)
 
+    def write_html(self, text, *args, **kwargs):
+        """Parse HTML and convert it to PDF"""
+        kwargs2 = vars(self)
+        # Method arguments must override class & instance attributes:
+        kwargs2.update(kwargs)
+        h2p = HTML2FPDF(self, *args, **kwargs2)
```

Comment:
Could you please preserve usage of `HTML2FPDF_CLASS`?
This class field has been documented here: https://pyfpdf.github.io/fpdf2/DocumentOutlineAndTableOfContents.html#with-html
Hence, it should still continue to work as it used to.
Optionnaly a `DeprecationWarning` could be raised, as this class attribute is not really useful anymore.

Reply:
In order to preserve the usage of `HTML2FPDF_CLASS`, I added it to the `FPDF` class directly instead of inheriting `HTMLMixin` class at e3d2b74e3fb21f13c079b12d674b6ef4c1104d3c.

<img width="562" alt="Screen Shot 2022-09-17 at 12 48 55 AM" src="https://user-images.githubusercontent.com/69574727/190846450-aef67998-9c81-45cb-ac04-d25f3aacea5c.png">

Reply:
Perfect, that's a good decision